src/components/SelectCircle.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import globalStyles from '../styles/styles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import themeColors from '../styles/themes/default';
+
+type SelectCircleProps = {
+    /** Should we show the checkmark inside the circle */
+    isChecked: boolean;
+
+    /** Additional styles to pass to SelectCircle */
+    styles: StyleProp<ViewStyle>;
+};
+
+function SelectCircle({isChecked = false, styles = []}: SelectCircleProps) {
+    return (
+        <View style={[globalStyles.selectCircle, globalStyles.alignSelfCenter, styles]}>
```

Comment:
Typescript made me get rid of the spread operator (`...styles` -> `styles`) after I switched from an `Array<>` to `StyleProp<>`. But the weird thing is that it still works when styles are passed like they used to be: `<SelectCircle isChecked={props.isSelected} styles={[styles.singleOptionSelectorCircle]} />`

Reply:
If it works, that's fine with me. Please test that the styles are displayed correctly.

Reply:
Yes, works just fine

Reply:
Should we check that styles isn't null?

Reply:
@cead22 Styles accepts falsy values, so no need for such check:
```ts
type Falsy = undefined | null | false;
export type StyleProp<T> =
  | T
  | RegisteredStyle<T>
  | RecursiveArray<T | RegisteredStyle<T> | Falsy>
  | Falsy;
```